src/kOS/Module/kOSProcessor.cs

```diff
@@ -240,11 +240,14 @@ public ModifierChangeWhen GetModuleCostChangeWhen()
             return ModifierChangeWhen.FIXED;
         }
 
+        public float additionalMass = 0F;
```

Comment:
How do you mean by "remain at the top" isn't it so if i put it at it's original place KSP will think it's the variable it needs for that field?